releases.json

```diff
@@ -48,7 +48,7 @@
             {
                 "device": "inf2",
                 "min_version": "0.0.22",
-                "max_version": "0.0.23",
+                "max_version": "0.0.24",
```

Comment:
Thanks, can you also update the releases section to include this new image?
```
"releases": [
        {
            "framework": "TGI",
            "device": "inf2",
            "version": "0.0.24",
            "os_version": "ubuntu22.04",
            "python_version": "py310",
            "pytorch_version": "2.1.2"
        }
]
```

Reply:
Sure! Thanks @Captainia.